src/main/java/me/zeroeightsix/fiber/constraint/ConstraintType.java

```diff
@@ -2,14 +2,69 @@
 
 import me.zeroeightsix.fiber.Identifier;
 
+/**
+ * A set of constraint types Fiber supports.
+ */
 public enum ConstraintType {
+	/**
+	 * Specifies a numerical lower bound.
+	 * <br> Values must be equal to or greater than the constraint's value to satisfy the constraint.
+	 */
 	NUMERICAL_LOWER_BOUND(true, identifier("min")),
+	/**
```

Comment:
This may be more of a style thing but it seems confusing that there is no empty line between the previous code and the next comment.